src/main/java/com/mixpanel/android/mpmetrics/MixpanelActivityLifecycleCallbacks.java

```diff
@@ -36,7 +35,9 @@ public void onActivityDestroyed(Activity activity) { }
     public void onActivitySaveInstanceState(Activity activity, Bundle outState) { }
 
     @Override
-    public void onActivityResumed(Activity activity) { }
+    public void onActivityResumed(Activity activity) {
```

Comment:
If AutoShowMixpanelUpdates is set to false, this function won't be registered. Am I right?